x/wasm/types/rent.go

```diff
@@ -0,0 +1,28 @@
+package types
+
+import (
+	"encoding/binary"
+	"errors"
+)
+
+// custom struct instead of protobuf to minimize state needed
```

Comment:
why not just use protobuf for consistency with other store objects?

Reply:
i was thinking that protobuf might be too heavy for such a simple struct but yeah i guess it probably won't be that bad. Better keep it consistent. Will change